src/Nox.Types.EntityFramework/Abstractions/NoxDatabaseConfigurator.cs

```diff
@@ -32,23 +33,111 @@ public abstract class NoxDatabaseConfigurator : INoxDatabaseConfigurator
             }
 
             // Override specific database provider configurators
+#pragma warning disable S3267 // Loops should be simplified with "LINQ" expressions
             foreach (var configurator in noxTypeDatabaseConfigurators)
             {
                 if (databaseProviderSpecificOverrides.IsInstanceOfType(configurator))
                 {
                     TypesDatabaseConfigurations[configurator.ForNoxType] = configurator;
                 }
             }
+#pragma warning restore S3267 // Loops should be simplified with "LINQ" expressions
         }
 
-        public virtual void ConfigureEntity(NoxSolutionCodeGeneratorState codeGeneratorState, EntityTypeBuilder builder,
-            Entity entity)
+        public virtual void ConfigureEntity(
+            NoxSolutionCodeGeneratorState codeGeneratorState,
+            EntityTypeBuilder builder,
+            Entity entity,
+            IReadOnlyList<RelationshipFullModel> relationships)
         {
-            //TODO Relations
-
             ConfigureKeys(codeGeneratorState, builder, entity);
 
             ConfigureAttributes(codeGeneratorState, builder, entity);
+
+            ConfigureRelationships(codeGeneratorState, builder, entity, relationships);
+        }
+
+        public virtual void ConfigureRelationships(
+            NoxSolutionCodeGeneratorState codeGeneratorState,
+            EntityTypeBuilder builder,
+            Entity entity,
+            IReadOnlyList<RelationshipFullModel> relationships)
+        {
+            var entityRelationships = relationships.Where(x => x.Entity.Name.Equals(entity.Name, StringComparison.InvariantCulture));
+
+            foreach (var fullRelationshipModel in entityRelationships)
+            {
+                builder
+                    .HasMany(fullRelationshipModel.RelationshipEntityType.FullName!, fullRelationshipModel.Relationship.Related.Entity.PluralName)
+                    .WithMany(fullRelationshipModel.Entity.PluralName);
+            }
+        }
+
+        public virtual List<RelationshipFullModel> GetRelationshipsToCreate(
```

Comment:
Configurator does not seem the right place  for this, should only care about configuring stuff